ql/src/java/org/apache/hadoop/hive/ql/plan/PartitionDesc.java

```diff
@@ -171,9 +171,9 @@ public String getDeserializerClassName() {
   public Deserializer getDeserializer(Configuration conf) throws Exception {
     Properties schema = getProperties();
     String clazzName = getDeserializerClassName();
-    Deserializer deserializer = ReflectionUtil.newInstance(conf.getClassByName(clazzName)
-        .asSubclass(Deserializer.class), conf);
-    SerDeUtils.initializeSerDe(deserializer, conf, getTableDesc().getProperties(), schema);
+    AbstractSerDe deserializer = ReflectionUtil.newInstance(conf.getClassByName(clazzName)
```

Comment:
Please call the variable serDe, and the function getSerDeClass, to be consistent with TableDesc.

Reply:
Done.